src/screens/Game/GameSubMenu/index.tsx

```diff
@@ -191,6 +222,53 @@ export default function GamesSubmenu({
                 {t('submenu.addShortcut', 'Add shortcut')}
               </button>
             )}
+            {isLinux && !eosOverlayInstalling && (
+              <button
+                className="link button is-text is-link"
+                onClick={async () => {
```

Comment:
This is definetelly doing a lot of stuff.
Move whats inside the onClick to a `useCallback` function like:
```
const handleEosOverlay = React.useCallback(async () => {
add rest of the function.

}, [add dependecies here])
```

then change the onclick to:
`onClick={handleEosOverlay}`